ironfish/src/network/peerNetwork.ts

```diff
@@ -408,7 +408,8 @@ export class PeerNetwork {
    * The full transaction will be sent to a subset of sqrt(num_peers)
    * and the rest of the peers will receive the transaction hash
    */
-  private broadcastTransaction(transaction: Transaction): void {
+  public broadcastTransaction(transaction: Transaction): void {
+    // TODO: is this change from private to public safe?
```

Comment:
If you want to figure out if its safe, you can look at wallet.broadcastTransaction to see what it does.